chart/compass/charts/director/config.yaml

```diff
@@ -85,6 +85,7 @@ graphql:
     deleteFormation: ["formation:write"]
     assignFormation: [ "formation:write" ]
     unassignFormation: [ "formation:write" ]
+    unassignFormationGlobal: [ "formation:global_write" ]
```

Comment:
Copy this in `components/director/hack/config-local.yaml` as well so that the mutation can be used in local run of director.